### PR TITLE
Italian translation.

### PR DESCRIPTION
**Pull request by teleyinex**

### CODE REVIEW

File: pybossa/default_settings.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 TERMSOFUSE = 'http://okfn.org/terms-of-use/'
 DATAUSE = 'http://opendatacommons.org/licenses/by/'
 LOGO = ''
-LOCALES = ['en', 'es']
+LOCALES = ['en', 'es', 'it']
```
Comment (alejandrodob) [inline pybossa/default_settings.py]: just one thing, why is not French here too?

Comment (teleyinex) [inline pybossa/default_settings.py]: Good one :-)

Comment (therealmarv) [inline pybossa/default_settings.py]: also wondering...

**Comment by alejandrodob** [comment]
:cake:

**Comment by therealmarv** [comment]
:cake: translation ok for current files. Can be merged. Needs update though after the merge of new Spanish and German files (new LOC in English). PR will follow very soon.

**Pull request merged**